cclib/parser/molcasparser.py

```diff
@@ -483,7 +483,8 @@ def extract(self, inputfile, line):
                 if line[1:48] == 'Molecular Partition Function and Molar Entropy:':
                     while 'TOTAL' not in line:
                         line = next(inputfile)
-                    entropy_values.append(utils.convertor(float(line.split()[2]), 'kcal/mol', 'hartree'))
+                    # Molcas reports entropy values in kcal/mol*K but actually writes them in cal/mol*K
+                    entropy_values.append(utils.convertor(float(line.split()[2]), 'kcal/mol', 'hartree') / 1000)
```

Comment:
Do we need to sum the entropy components for Moclas as well?

Reply:
I don't think so. The output of Molcas produces values for a set of temperatures, we end up getting the entropy value for 298.15 K in https://github.com/cclib/cclib/blob/b50fdfd557077e9dfbd3d1605c07b5f82d90fb04/cclib/parser/molcasparser.py#L495-L513

Is that what you mean?

Reply:
I see. And is there something we can do if 298.15 is not there but other temperature are?

Reply:
Not sure, I've never used Molcas. Is it possible that 298.15 K is always reported?

Reply:
I guess the comment says it all. Looks fine as far as your changes are concerned.